Xcore_Local.podspec

```diff
@@ -11,8 +11,7 @@ Pod::Spec.new do |s|
   s.requires_arc          = true
   s.ios.deployment_target = '9.0'
   s.pod_target_xcconfig   = {
-    'SWIFT_VERSION' => '3.0',
-    'OTHER_SWIFT_FLAGS' => '-DXCORE_ENVIRONMENT_${CONFIGURATION}'
```

Comment:
let's revert this please this is just for local example project. Right now the example project is not showcasing the use-case of `Environment` class.

Reply:
that line is actually breaking the build when I check it out from cocoapods.

Reply:
`Xcore_Local.podspec` this is not public podspec this isn't pulled to the projects